### PR TITLE
Fixes, bumps, eslint

### PR DESCRIPTION
**Pull request by mistadikay**
- fixed logo not working due to the breaking changes between webpack `2.1.0-beta.5` and `2.1.0-beta.25`
- updated all dependencies to the last versions and fixed some regressions due to API changes
- replaced `jscs` and `jshint` with `eslint`. I used [standard.js](http://standardjs.com/) config with a few changes, but we should move to [our own config](https://github.com/zero-plus-x/eslint-config-0x/) as soon as it'll be updated (follow [this issue](https://github.com/zero-plus-x/eslint-config-0x/issues/1) for the updates)
- a few other minor fixes

There are still lots of things that can be refactored in this project — I fixed only the most obvious stuff, so we can already use this logo if needed.

### CODE REVIEW

**Comment by mistadikay** [comment]
@yuri-karadzhov or @nkabardin for code review

File: src/logo/logo.js [hunk 1]
```diff
@@ -3,23 +3,23 @@
 import logo from '../models/logo-flat.json';
 import particle from '../textures/particle.png';
 
-import THREE from 'three';
+import * as THREE from 'three';
```
Comment (yuri-karadzhov) [inline src/logo/logo.js:4]: Why do we need \* as here? Do they delete default export?

File: webpack.config.babel.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 'use strict';
```
Comment (yuri-karadzhov) [inline webpack.config.babel.js]: unnecessary 'use strict'

File: webpack.config.babel.js [hunk 3]
```diff
@@ -9,31 +10,54 @@ import HtmlWebpackPlugin from 'html-webpack-plugin';
 import cfg from './build.config';
 
 const IS_DEVELOPMENT = process.env.NODE_ENV !== 'production';
-
+const contextPath = path.join(__dirname, 'src');
+const outputPath = path.join(__dirname, 'dist');
```
Comment (yuri-karadzhov) [inline webpack.config.babel.js]: It's better to move to build.config.js so you will have:

```
const contextPath = path.join(__dirname, cfg.src);
```

File: webpack.config.babel.js [hunk 4]
```diff
@@ -45,47 +69,31 @@ const config = {
   module: {
     loaders: [{
       test: /\.scss$/,
-      include: `${__dirname}/src`,
-      loader: IS_DEVELOPMENT ?
-        'style!css!resolve-url!sass' :
-        ExtractTextPlugin.extract('css?minimize!postcss!resolve-url!sass')
+      include: path.join(__dirname, 'src'),
```
Comment (yuri-karadzhov) [inline webpack.config.babel.js]: Can reuse `contextPath` and rename it `srcPath`

File: webpack.config.babel.js [hunk 5]
```diff
@@ -45,47 +69,31 @@ const config = {
...
-        `${__dirname}/src/models`
-      ],
+      include: path.join(__dirname, 'src/models'),
       loader: 'file-loader?name=[path][name].[ext]'
     }, {
       test: /\.json/,
-      include: [
-        `${__dirname}/src/fonts`
-      ],
+      include: path.join(__dirname, 'src/fonts'),
```
Comment (yuri-karadzhov) [inline webpack.config.babel.js]: It's good to move all paths to build.config.js

File: webpack.config.babel.js [hunk 6]
```diff
@@ -95,22 +103,29 @@ const config = {
   }
 };
 
-if(!IS_DEVELOPMENT) {
+if (!IS_DEVELOPMENT) {
   config.module.loaders.push({
     test: /\.js$/,
-    include: `${__dirname}/src`,
+    include: path.join(__dirname, 'src'),
```
Comment (yuri-karadzhov) [inline webpack.config.babel.js]: Can use `srcPath` once again

**Review by yuri-karadzhov — COMMENTED** [review]
Good config update. I have pretty much the same for my current projects (except some eslint rules)

File: src/logo/logo.js (hunk 1 above)
Comment (mistadikay) [reply src/logo/logo.js:4]: yes, they moved to ES6 modules in r80 and there is no default export anymore

File: webpack.config.babel.js (hunk 2 above)
Comment (mistadikay) [reply webpack.config.babel.js]: removed in https://github.com/zero-plus-x/interactive-logo/pull/1/commits/c360687f8769f4753655e6b1e2eaf1ea8e4f26da

File: src/logo/logo.js (hunk 1 above)
Comment (yuri-karadzhov) [reply src/logo/logo.js:4]: Do we need all THREE at once, probably we can import only stuff we need then. Actually it's even better to remove threejs dependency at all (cause it's only used for providing projection matrix). But we should do one sing at a time of course.

Comment (mistadikay) [reply src/logo/logo.js:4]: perhaps not, I think we can do some refactoring in another branch later, this one was intended only to resurrect the project and fix a few bugs. Let's do on thing a time, yeah. :)

File: webpack.config.babel.js (hunk 3 above)
Comment (mistadikay) [reply webpack.config.babel.js]: done in https://github.com/zero-plus-x/interactive-logo/pull/1/commits/78ac912ba1b7f53f4aa3f61565b753184a1b0cdc So, I think we can merge this branch and do a more thorough refactoring later as a separate PR

**Comment by yuri-karadzhov** [comment]
👍

**Pull request merged**